### PR TITLE
Get User Data

### PR DESCRIPTION
**Pull request by akoredetunde**
I created a function for the auth microservice that get user data from the file storage service

### CODE REVIEW

File: utils/getUserData.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+async function getUserData(endPoint, username){
+  //   endPoint ---- string ----- always add a forward slash / in front of the endpoint url
+  //   username ------ string
+  // dynamic url
+  let url = endPoint + username;
```
Comment (jriffs) [inline utils/getUserData.js:5]: okay I see what you're trying to achieve here, but I feel using template strings would be a better way of constructing the url. This 👇 would've been better
```
let url = `/${endpoint}/${username}`
```

File: utils/getUserData.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  //   endPoint ---- string ----- always add a forward slash / in front of the endpoint url
+  //   username ------ string
+  // dynamic url
+  let url = endPoint + username;
+  let option = {
+    method: "GET",
+    headers:{
+      Accept: "*",
+    },
+    Origin: "auth"
```
Comment (jriffs) [inline utils/getUserData.js:11]: This should be "originator", and should be in the `headers` object. we're basically applying our own request header, so don't let it confuse you.
This 👇 what I mean.
```
headers:{
      Accept: "*",
      originator: "auth"
    }
```

**Review by jriffs — CHANGES_REQUESTED** [review]
Great work bro 💪... just make those changes and we'll be good to go 👍

File: utils/getUserData.js (hunk 1 above)
Comment (akoredetunde) [reply utils/getUserData.js]: On it bro

File: utils/getUserData.js [hunk 3]
```diff
@@ -1,14 +1,12 @@
 async function getUserData(endPoint, username){
-  //   endPoint ---- string ----- always add a forward slash / in front of the endpoint url
-  //   username ------ string
   // dynamic url
-  let url = endPoint + username;
+  let url = "/${endPoint}/${username}"
```
Comment (jriffs) [inline utils/getUserData.js:3]: Use Template literals .. 
```
this `` instead of this ""
```

**Review by jriffs — CHANGES_REQUESTED** [review]
another change required bro

File: utils/getUserData.js (hunk 3 above)
Comment (akoredetunde) [reply utils/getUserData.js]: oh, pardon forgot I'm using template literals.

**Review by jriffs — APPROVED** [review]
Nice .. great work bro

**Pull request merged**